### DISCUSSION TITLE
A sneak-peek of my chat app: "Lightable" (Desktop made in 100% gopher)

### DISCUSSION THREAD

**Discussion by brys0** [wailsapp/wails#1738] [Show and tell]
Hey! I'd love to post my project to the showcase. Unfortunately, there's still much work to do. Here's some pages and such though. 

## Preview
![Home Page](https://github.com/brys0/Lightable/blob/pre-alpha/assets/Lightable_Red-dev_eH4nN3zgTK.png?raw=true)
![Home Page (With Prev User)](https://github.com/brys0/Lightable/blob/pre-alpha/assets/Lightable_Red-dev_HAiitHwuzn.png?raw=true)
![Coming Soon Concept](https://github.com/brys0/Lightable/blob/pre-alpha/assets/Lightable_Red-dev_e8R25vJF8m.png?raw=true)
![Early Access Window](https://github.com/brys0/Lightable/blob/pre-alpha/assets/Lightable_Red-dev_058Ky6HKv7.png?raw=true)
![0.0.5 ALpha Main Page](https://github.com/brys0/Lightable/blob/pre-alpha/assets/Lightable_Red-dev_jmzHaS6Ke9.png?raw=true)

Mono Repo (If your interested) -> https://github.com/brys0/Lightable

The go code probably isn't great, my apologies. If you want to fix some things, or have an issue with the ways things are laid out. Feel free to make a PR! 

This project is going to quite *big* for a junior dev like me, its alot, balancing work (and soon to be school) it will take much longer to release features. 

This project has been going on for awhile. (With many different names) but I think I'm happy with this implementation. 

I've learnt alot. yet still want to learn more. I look forward to posting future updates here. And maybe one day.. a production version.

Who knows, all I know is that I was proud enough to post this. So if you like it let me know! I'd love to see some feedback. 
The homepage is a bit rough.. I'm still working on a design I like.

**Comment by leaanthony** [comment]
Nice work! Keep going!

**Reply by brys0** [reply]
Much love. I'll see what I can do!
